includes/PhpStan/ServicesDynamicReturnTypeExtension.php

```diff
@@ -0,0 +1,98 @@
+<?php
```

Comment:
PHPStan files should live in `tests/phpstan`, never in `includes/`

Reply:
Fixed